CONTRIBUTING.md

```diff
@@ -80,7 +80,7 @@ any issues they report.
 Then run the framework's automated tests with the following command:
 
 ```shell
-pytest --ignore=tests/test_examples
+pytest --ignore=tests/integration/examples
```

Comment:
`pytest tests/unit` should be simplerr?

Reply:
Yes, my bad. The `unit2` folder persisted locally despite being absent on the `development` branch on GitHub. The most recent commit should fix this.